source/common/grpc/common.cc

```diff
@@ -53,11 +54,12 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   }
   cluster.statsScope()
       .counter(fmt::format("{}.{}.{}.{}", protocol, grpc_service, grpc_method,
-                           grpc_status->value().c_str()))
+                           grpc_status->value().getStringView()))
       .inc();
   uint64_t grpc_status_code;
+  std::string grpc_status_string(grpc_status->value().getStringView());
   const bool success =
-      StringUtil::atoull(grpc_status->value().c_str(), grpc_status_code) && grpc_status_code == 0;
+      StringUtil::atoull(grpc_status_string.c_str(), grpc_status_code) && grpc_status_code == 0;
```

Comment:
Optimally `atoull` would take a string view. Can you TODO this for a follow up? Same elsewhere.